matcher/service/openreview_interface.py

```diff
@@ -74,7 +74,7 @@ def _edge_to_score(edge, translate_map=None):
 
     return score
 
-def get_all_edges(client, edge_invitation_id, logger=None):
+def get_all_edges(client, edge_invitation_id, all_papers, all_reviewers, logger=None):
```

Comment:
This method can go inside of `ConfigNoteInterface` so you don't need to pass the papers and reviewers every time.